docs/source/tutorial.rst

```diff
@@ -44,6 +44,11 @@ Let's access the URL of the created object:
 Getting access
 --------------
 
+There two ways of getting access to contract but soon the first method can be deprecated. Thus, better consider using the second method. Link to it you can find above.
```

Comment:
There are two mechanisms of getting access to contract.

Note that the first mechanism will be deprecated soon. Thus, it is recommended to use the second one (see information below).